src/core/lib/experiments/experiments.yaml

```diff
@@ -91,6 +91,11 @@
   expiry: 2024/12/01
   owner: alishananda@google.com
   test_tags: [resource_quota_test]
+- name: local_connector_secure
+  description: Local Security Connector uses TSI_PRIVACY_AND_INTEGRITY only for UDS.
```

Comment:
Might be a good idea for this to be re-worded to indicate what's actually changing:

Local security connector uses TSI_SECURITY_NONE for LOCAL_TCP connections.